src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -483,6 +556,11 @@ Engine::LoadExeNetworkImpl(const InferenceEngine::CNNNetwork &network, const std
     // TODO: Clarify the behavior of SetConfig method. Skip eng_config or not?
     Config conf = engConfig;
 
+    // save hints parameters to model rt_info
+    if(is_cpu_map_available()) {
```

Comment:
I supposed we don't need this condition, do we?

Reply:
for threading 1.0, saving hints parameters is performed in function "ApplyPerformanceHints", so I think here this condition is needed and I will add condition for loading caching configs.

Reply:
Will remove "is_cpu_map_available()" after Xiaoxia merge all PRs for MT 2.0 flow.